### PR TITLE
The next web

### PR DESCRIPTION
**Pull request by carloso0114**
This project consists of replicating the The Next Web website putting emphasis on how it behaves differently depending on the size of the screen where the website is rendered.
With the use many media="screen and (min-width: 768px) and (max-width: 1023px)" i was able to manage 3 css files of the same html, giving apropiate styles for each screen sizes or resolutions up to 1920x1080p.
The idea is to give the clone the break points from the original page with different layout.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,503 @@
...
+      </section>
+      <section class="latest-funding-rounds">
+        <header class="index-co">
+          <h2 class="inline "> <a href="#"> Latest funding rounds on Index.co </a></h2>
+          <div class="index-links">
+            <a href="#" class="float-link inline ">See more</a>
+            <a href="#" class="float-link inline">Get updates by email </a>
+            <a href="#" class="float-link inline">Add info </a>
+          </div>
+        </header>
```
Comment (jubaan) [inline index.html]: - [x] The header should be outside the `<main>` tag, and be around the `<nav>` tag. The information here that is inside the `<header>` should stay in this position with a suitable HTML tag.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,503 @@
...
+  <link rel="stylesheet" media="screen and (min-width: 768px) and (max-width: 1023px)" href="tablet.css">
+  <link rel="stylesheet" media="screen and (max-width: 767px)" href="phone.css">
+  <script src="https://use.fontawesome.com/6a167e652a.js"></script>
+  <script src="https://unpkg.com/ionicons@5.2.1/dist/ionicons.js"></script>
+</head>
+<body>
+  <div class="nav-grid">
+    <div class="nav-logo">
+      <a href="#"><img src="assets/logo.svg" alt="logo"></a>
+    </div>
```
Comment (jubaan) [inline index.html:17]: - [x] This should be included in the navbar

File: phone.css [hunk 3]
```diff
@@ -0,0 +1,612 @@
...
+}
+
+.primary-nav li a {
+  color: #858585;
+  text-decoration: none;
+}
+
+.highlight {
+  color: white !important;
+}
```
Comment (jubaan) [inline phone.css]: - [x] Remove the `!important` and review the specificity or the position of the selector, maybe another property in a later selector is overriding the application of the color white.

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

## :wave: :wave: Hello @carloso0114  . Thanks for submitting you project!

Great thing this project has and has done well:
:100: The project has a navbar that has all the needed elements and changes accordingly on different screen-sizes
:100: Has a 3 image section set as background, and changes accordingly on different screen-sizes
:100: Has latest funding section that modifies layout accordingly to the different screen-sizes
:100: It has "Other Categories section" that change layout accordingly to the different screen-sizes
:100: Footer has all elements
:100: the website is using at least 2 media queries
:100: you're using flex, float, and grid to align elements 

There are some issues that need to be addressed before we can move forward. Please review the list below.

## Required changes :wrench: 
### [Mandatory] :red_circle: 

- [x] Instructions in the README on **How to Run int** ask the user to clone a different repository.

- [x] The GitHub link in the Author's social links redirects to another repository. Please make it a link to the user profile.

- [x] The footer doesn't change on mobile version as original website. 
image

- [x] The footer is missing a TNW logo as the original website

- [x] Use [semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp) for the `<header>`. See inline comment

- [x] The `<header>` should wrap the `<nav>`

- [x] The `<nav>` should wrap the logo. See inline comment

- [x] The use of `!important` is discouraged. Please remove it from all the CSS files.

- [x] The 3 image section is missing information
image

- [x] Image look distorted.
image

- [x] Element is not well positioned in lates deals section
image


### [OPTIONAL]  :yellow_circle:  

- [ ] Change **Lorem Ipsum** to words that mimic the original website.

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Review by Srjnnnn — CHANGES_REQUESTED** [review]
**### Status: Changes Required :recycle:** 
Hey @carloso0114 
You've done a great job! Keep up the good work.
I would like to mention that this project took a lot of time for me. It takes time and some reviews to reach the perfect version. Believe me, it's also good to have a nice project at your GitHub profile :+1: 
General comment:
I highly recommend trying to mimic the page as much as you can. This will be your showcase for the job applications so it's better to get the necessary sentences from the page. I checked and just found one media query and it is for min-width 322px. I am adding the screenshot from pathwright and kindly find the necessary breakpoints. (Hint: try to use developer console and check the different UI options from the bar at the top)
Also, I've realized that you've already found the media breakpoints but I've searched all the CSS files and can't find the necessary media queries!

six

For the medium Screen
two
Your see more, get updates and add info should be on one line.

four
Right now your latest deals is looking like above. Please widened it to have all the width with some margin. Kindly check the original.

For the Large Screen: 
Optional:
It would be nice to have some writings on the leftmost part of the image.
Also if you add some sentences to the left ones as well, this will look better and closer to the original.
one
Mandatory: 
Your Lorem ipsum is behind the heading and if you change it as well that would be cool.

If you've questions about the review, you can reach me via Slack. :100: 

When you done with the requested changes, please submit another code review. Good luck. :crossed_fingers:

**Review by jubaan — APPROVED** [review]
# :tada: :tada: :tada: :rotating_light: Status: Approved :rotating_light:  :tada: :tada: :tada:

#### :clap: :clap: Great job :hammer_and_pick:!! There are no more issues left to be addressed. 

## See you at the top :rocket:!

# :tada: :tada: :tada: :rotating_light: Status: Approved :rotating_light:  :tada: :tada: :tada:

**Pull request merged**